SandboxOrchestrationPackage/DataModel/datamodel.xml

```diff
@@ -2,16 +2,16 @@
 	<ns0:ResourceFamilies>
 	</ns0:ResourceFamilies>
   <ns0:ScriptDescriptors>
-    <ns0:ScriptDescriptor Alias="Setup script" CategoryName="Hidden Commands" Name="Default Sandbox Setup 3.0" Version="3.5.0">
+    <ns0:ScriptDescriptor Alias="Setup script" CategoryName="Hidden Commands" Name="Default Sandbox Setup 3.0" Version="4.0.0">
```

Comment:
If we aren't going to release a new sandbox orchestration scripts then this change might be redundant, right? (talking about all four scripts)

Reply:
Sveta talked with Tomer A. and we are not releasing new sandbox orchestration scripts. This change is reverted.